### ISSUE TITLE
Resource Editor is slow to populate.

### ISSUE THREAD

**Issue by AdamSpeight2008** [dotnet/project-system#4790]
Opening a large resx for example [VBResources.resx](https://github.com/dotnet/roslyn/blob/master/src/Compilers/VisualBasic/Portable/VBResources.resx) takes an inordinately long time to populate the window and listview. Giving the impression the UI has crashed.  

Version used.
```
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.1.0 Preview 2.0
VisualStudio.16.Preview/16.1.0-pre.2.0+28822.285
Microsoft .NET Framework
Version 4.7.03190
```
Also tried building from source and running that, still slow.

**Comment by Pilchie** [comment]
Try re-sizing the designer. There is a bug in Winforms in 4.7 and above that is causing the page to not draw initially.  The team is investigating a workaround.

**Comment by AdamSpeight2008** [comment]
@Pilchie Not a great workaround but it sort of works.

**Comment by Pilchie** [comment]
Agreed - we're working on a fix in the product for 16.1
